frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -363,18 +372,23 @@ local option_titles = {
     all_at_once = _("Show all at once"),
     reclaim_height = _("Reclaim bar height from bottom margin"),
     toc_markers = _("Show chapter markers"),
-    page_progress = _("Current page"),
-    time = _("Current time"),
-    pages_left = _("Pages left in chapter"),
-    battery = _("Battery status"),
-    percentage = _("Progress percentage"),
-    book_time_to_read = _("Book time to read"),
-    chapter_time_to_read = _("Chapter time to read"),
-    frontlight = _("Frontlight level"),
-    mem_usage = _("KOReader memory usage"),
-    wifi_status = _("Wi-Fi status"),
+    page_progress = _("Current page") .. " (/)",
```

Comment:
Why concatenation? This way a translator can't reverse the order if desired. The string will show up as a 95 % match suggestion so minor changes aren't generally a big problem.

Reply:
done